src/Provider.tsx

```diff
@@ -100,6 +127,20 @@ function reducer(state: State, action: Action): State {
         tabStops
       };
     }
+    case ActionTypes.TAB_TO_FIRST:
```

Comment:
This supposed to be empty?

Reply:
It's not empty. `TAB_TO_FIRST` and `TAB_TO_LAST`  use the same block. I've copied the switch-fallthrough style from `TAB_TO_PREVIOUS`/`TAB_TO_NEXT`.